app/src/main/res/layout/continue_interaction_item.xml

```diff
@@ -46,14 +50,19 @@
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent" />
 
-    <Button
-      android:id="@+id/continue_button"
+    <org.oppia.android.app.customview.ContinueButtonView
+      android:id="@+id/animation_continue_button"
       style="@style/StateButtonActive"
-      android:layout_width="wrap_content"
+      android:layout_marginEnd="10dp"
```

Comment:
Why is this needed now?

Reply:
PTAL at the above image

Reply:
Closing this to consolidate to a single thread that covers "why are there extra changes needed."